### DISCUSSION TITLE
Map dir/ URL to dir/index.md

### DISCUSSION THREAD

**Discussion by alkisg** [facebook/docusaurus#6021] [Q&A] [answered]
Hi, I have the following structure:

- docs/index.md
- docs/page1.md
- docs/page2.md

So I can navigate to https://example.com/docs/index, but when I go to https://example.com/docs/ I get "Page not found".
The same thing happens if I go e.g. to https://docusaurus.io/docs/i18n/.

Isn't there a way to map a directory URL to its child index.md?

I only started using docusaurys an hour ago so I might be missing something obvious! Thanks!

**Comment by Josh-Cena** [comment, marked as answer]
You can use the `slug` front matter to customize the routing: https://docusaurus.io/docs/docs-introduction#home-page-docs So you can set `slug: /` for the doc that you want to appear at `https://example.com/docs/`. It's not guaranteed that if `https://example.com/docs/foo/bar/` exists, `https://example.com/docs/foo/` must exist.

**Reply by slorber** [reply]
Also note that we'll likely make `index.md` use `/` as route very soon (track https://github.com/facebook/docusaurus/issues/2537)

**Comment by alkisg** [comment]
Thank you very much!
